SportsHub/SportsHub.Core/Models/Validations/UserValidator.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using FluentValidation;
+
+namespace SportsHub.Domain.Models.Validations
+{
+    public class UserValidator : AbstractValidator<User>
```

Comment:
Fluent Validator for validation of API requests. Move it to the API level

Reply:
The Fluent Validation is now moved to the API level.